src/ResourceManager/KeyVault/Commands.KeyVault/Commands/RemoveAzureKeyVaultAccessPolicy.cs

```diff
@@ -109,25 +205,44 @@ public class RemoveAzureKeyVaultAccessPolicy : KeyVaultManagementCmdletBase
             ParameterSetName = ByObjectId,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "Specifies the ID of application that a user must.")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = InputObjectByObjectId,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Specifies the ID of application that a user must.")]
```

Comment:
'whose permissions should be removed'. (Again, not your bug, but while you're in there..)